docs/reference/functional-testing.md

```diff
@@ -2,19 +2,40 @@
 title: "Functional Tests"
 ---
 
-Current as of `March 11th, 2021`
+Current as of `October, 2023`
 
-End to end testing of the entire FxA ecosystem is provided by a [comprehensive suite of Selenium tests](https://github.com/mozilla/fxa/tree/main/packages/fxa-content-server/tests/functional) in the fxa-content-server package. Tests can be run by going to the content-server package directory and typing:
+End to end functional testing of the entire FxA ecosystem is achieved by a [comprehensive suite of Playwright tests](https://github.com/mozilla/fxa/tree/main/packages/functional-tests/tests). These tests are situated within the `Packages` directory and are organized into distinct folders corresponding to their respective functionality, such as sign-in, sync, OAuth, and others. These tests can be run through the packages directory using the below command:
 
 ```bash
-$ npm run test-functional
+$ yarn workspace functional-tests test
 ```
 
-The full set of functional tests is run on [CircleCI](./tests-in-circleci) on every checkin and every time a pull request is merged to main.
-This full set consist of a smoke test suite(https://github.com/mozilla/fxa/tree/main/packages/fxa-content-server/tests/functional_smoke) which runs the high priority test cases first and upon success the full suite of regression tests(https://github.com/mozilla/fxa/tree/main/packages/fxa-content-server/tests/functional_regression) are run. If there is a failure in the smoke test suite, the regression suite won't be run until the failures have been fixed.
-There is also a notification system in place for when these failures occur to alert the FxA team via Slack messaging.
+The full set of functional tests is run on [CircleCI](./tests-in-circleci) on every check in and every time a pull request is merged to main.
+The High priority tests are tagged as Severity 1 tests, medium priority as Severity 2 and low priority tests are non-tagged. The Production smoke test suite comprises all the Severity 1 (High Priority) tests which are run during the production deployments. Similarly, all the Severity 1 (High Priority) and Severity 2 (Medium Priority) tests make up the Stage smoke test suite, run during the Stage deployments.
+There is also a notification system in place for when there is a stage or prod smoke test suite failure to alert the FxA team via Slack messaging.
 
-[The Intern](https://theintern.io/) library is used to run the tests, which itself is a wrapper around the [Leadfoot](https://theintern.io/docs.html#Leadfoot/2/api/Command) WebDriver library.
+## How to decide priority (high, medium, low) for a test?
+Deciding the priority of tests is a crucial part of test management. The priority of a test case can vary depending on factors such as its impact on the application, its criticality, and the frequency of execution. Here's how you can decide the priority of your tests:
+
+* High Priority test
```

Comment:
I'm not sure how this will render.  We might want to make these sub-headers (e.g. ### or ####)

Reply:
It should be a bullet point.